pyvisa-py/protocols/usbtmc.py

```diff
@@ -115,17 +115,27 @@ class BulkInMessage(namedtuple('BulkInMessage', 'msgid btag btaginverse '
     def from_bytes(cls, data):
         msgid, btag, btaginverse = struct.unpack_from('BBBx', data)
         if msgid != MsgID.dev_dep_msg_in:
-            raise ValueError("Unexpected MsgID 0x{:02x}".format(msgid))
+                        return BulkInMessage.from_quirky(data)
```

Comment:
The indentation is incorrect and furthermore since this is a bug of the instrument, I would suggest we raise a UserWarning suggesting the user to update the firmware or something similar.

Reply:
@twam